### PR TITLE
generic: 5.15: multiple cleanup for upstream patch

### PR DESCRIPTION
**Pull request by Ansuel**
While start working on 6.1 support it was notice that many patch got merged upstream and can be moved from pending to backport.
Move each patch that got merged upstream from pending to backport with the correct tag.

### CODE REVIEW

**Comment by neheb** [comment]
```
Subject: [PATCH 1/1]
```

hmmmm.

**Comment by Ansuel** [comment]
@neheb patch are generated from git format-patch -n1 on the upstream patch so this is not a simple move

**Comment by neheb** [comment]
weird. git format-patch -1 doesn't add 1/1.

https://github.com/torvalds/linux/$PATCH.patch doesn't either.

**Comment by Ansuel** [comment]
It's probably a stupid thing of using -n1 instead of -1... didn't know -1 was even a thing

**Comment by neheb** [comment]
@Ansuel haha I didn't know -n1 was a thing :).

**Comment by Ansuel** [comment]
Anyway if we really want i can refresh them using -1 but it's really a formal thing ahahha

**Comment by neheb** [comment]
right. just an oddity. From what I've seen the github line above usually has the least amount of extra stuff. Of course quillt will clean most of it.

File: target/linux/generic/backport-5.15/201-v5.16-scripts-dtc-Update-to-upstream-version-v1.6.1-19-g0a.patch [hunk 1]
```diff
@@ -0,0 +1,997 @@
...
++
++	irq_map_prop = get_property(node, "interrupt-map");
++	if (!irq_map_prop)
++		return;
++
++	if (node->addr_cells < 0) {
+ 		FAIL(c, dti, node,
+-		     "Missing #address-cells in interrupt provider");
++		     "Missing '#address-cells' in interrupt-map provider");
++		return;
```
Comment (john-tho) [inline target/linux/generic/backport-5.15/201-v5.16-scripts-dtc-Update-to-upstream-version-v1.6.1-19-g0a.patch:347]: This was were the Openwrt pending change [commit in dtc/dtc](https://git.kernel.org/pub/scm/utils/dtc/dtc.git/commit/?id=d8d1a9a77863a8c7031ae82a1d461aa78eb72a7b) happened, but the linux upstream DTC roundup changes commit includes an awful lot more. Would need to test, as well as mention in commit if we include the whole roundup.

File: target/linux/generic/backport-5.15/344-v5.18-02-phy-marvell-phy-mvebu-a3700-comphy-Add-native-kernel.patch [hunk 2]
```diff
@@ -1197,21 +1182,10 @@ Signed-off-by: Marek Behún <kabel@kernel.org>
...
-+	return 0;
-+}
-+
-+static void
-+mvebu_a3700_comphy_usb3_power_off(struct mvebu_a3700_comphy_lane *lane)
-+{
-+	/*
-+	 * Currently the USB3 MAC sets the USB3 PHY to low state, so we do not
-+	 * need to power off USB3 PHY again.
-+	 */
```
Comment (john-tho) [inline target/linux/generic/backport-5.15/344-v5.18-02-phy-marvell-phy-mvebu-a3700-comphy-Add-native-kernel.patch:1214]: This cut section is from the (now missing) fix:
https://github.com/torvalds/linux/commit/0a6fc70d76bddf98278af2ac000379c82aec8f11

File: target/linux/generic/backport-5.15/201-v5.16-scripts-dtc-Update-to-upstream-version-v1.6.1-19-g0a.patch (hunk 1 above)
Comment (Ansuel) [reply target/linux/generic/backport-5.15/201-v5.16-scripts-dtc-Update-to-upstream-version-v1.6.1-19-g0a.patch:347]: they are dtc bump should not hurt to have more check for dtc in theory, the idea is to have as little changes as possible for backport patch

File: target/linux/generic/backport-5.15/344-v5.18-02-phy-marvell-phy-mvebu-a3700-comphy-Add-native-kernel.patch (hunk 2 above)
Comment (Ansuel) [reply target/linux/generic/backport-5.15/344-v5.18-02-phy-marvell-phy-mvebu-a3700-comphy-Add-native-kernel.patch:1214]: so we need to include another patch

**Comment by hauke** [comment]
This looks good, but I have to look into the patches in more detail. You probably found some of these while working on kernel 6.1.

You can also use the `-k` option for `git format-patch` to not emit a `[PATCH]` at all, but I am fine if you leave them like they are now.

**Comment by Ansuel** [comment]
@hauke sure take your time to check this... yes found these with the 6.1 work and seems a nice idea to clean 5.15 now instead of checking in 6.1 if the backport is clean...

there are also 2 backport patch that are not actually backport and should be placed in pending but they will come later i think

File: target/linux/generic/backport-5.15/783-v6.1-net-sfp-re-implement-soft-state-polling-setup.patch [hunk 3]
```diff
@@ -0,0 +1,98 @@
+From 8475c4b70b040f9d8cbc308100f2c4d865f810b3 Mon Sep 17 00:00:00 2001
+From: "Russell King (Oracle)" <rmk+kernel@armlinux.org.uk>
+Date: Tue, 13 Sep 2022 20:06:27 +0100
+Subject: [PATCH 1/1] net: sfp: re-implement soft state polling setup
```
[the remark this answers is not in the document]
Comment (Ansuel) [reply target/linux/generic/backport-5.15/783-v6.1-net-sfp-re-implement-soft-state-polling-setup.patch:4]: i notice they changed the order so i decided to split them

**Review by hauke — APPROVED** [review]
Looks good to me. 
Thanks for moving the patches which are actually not backported to pending.

**Pull request merged**